packages/flutter/lib/src/painting/box_painter.dart

```diff
@@ -1127,6 +1115,114 @@ enum BoxShape {
   circle
 }
 
+/// An immutable set of radii for each corner of a rectangle. Only used with
+/// [BoxShape.rectangle].
+class BorderRadius {
```

Comment:
I'm not really asking for a name change here (or elsewhere) but in this case CornerRadii might have been more apt.